docs/wdc_library_versions.md

```diff
@@ -38,8 +38,8 @@ The following table displays which versions of the WDC are compatible with Table
 |`tableauwdc-2.2.0`    |10.2 or later|
 |`tableauwdc-2.1.0`    |10.1 or later|
 |`tableauwdc-2.0.2`    |10.0 and later |
-|`tableauwdc-2.0.0`    |10.0 through 2019.1 |
-|`tableauwdc-1.1.1`    |9.3 through 2019.1<br />9.2.4 through 2019.1 <br />9.1.6 through 2019.1|
+|`tableauwdc-2.0.0`    |10.0 through 2019.2 |
```

Comment:
as above (and below), this isn't necessarily true...

Reply:
Yes. That's true. I don't have a good fix for this until we know which version will get WebEngine.  For now, I'll just have this say "... through 2019.2"  - while technically true, it doesn't tell the whole story.